src/components/Rating/RatingCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import defaultAvatar from '../../assets/images/avatar.png';
+
+const RatingCard = ({ rate }) => (
+  <div className="rating-card">
+    <div className="rating-card__left">
+      <div className="rating">
+        <button className={rate.rating === 5 ? 'selected' : ''} />
+        <button className={rate.rating === 4 ? 'selected' : ''} />
+        <button title="text" className={rate.rating === 3 ? 'selected' : ''} />
```

Comment:
@NkFab Are you using the title's attribute anywhere? If not, it should be removed as it is not part of the `button's tag` attributes.

Reply:
Done